packages/itwinui-css/src/button/status-color.scss

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+// See LICENSE.md in the project root for license terms and full copyright notice.
+@import '../style/index';
+@import './button';
+
+@mixin iui-button-status-color($button-color) {
```

Comment:
Renamed the mixin to `iui-button-variant` & the file name to match.

I did not make the change yet to accept `high-visibility` or `cta` as the args yet.

Reply:
I've updated `variant.scss` to accept the button variant name as `@arg` instead of color.  Personally I think the older way was cleaner code wise but the `@arg`s make more sense this new way.